python/lsst/ts/criopy/m1m3/force_calculator.py

```diff
@@ -124,28 +124,72 @@ class AppliedForces:
             Total force. Square root (
         """
 
+        xForces = [0.0] * FATABLE_XFA
+        yForces = [0.0] * FATABLE_YFA
+        zForces = [0.0] * FATABLE_ZFA
+        fx = 0.0
+        fy = 0.0
+        fz = 0.0
+        mx = 0.0
+        my = 0.0
+        mz = 0.0
+        forceMagnitude = 0.0
+
         def __init__(
             self,
-            x_forces: list[float],
-            y_forces: list[float],
-            z_forces: list[float],
+            x_forces: list[float] | None = None,
+            y_forces: list[float] | None = None,
+            z_forces: list[float] | None = None,
             fas: dict[str, Any] | None = None,
         ):
+            if x_forces is None:
+                x_forces = [0.0] * FATABLE_XFA
+            if y_forces is None:
+                y_forces = [0.0] * FATABLE_YFA
+            if z_forces is None:
+                z_forces = [0.0] * FATABLE_ZFA
+
             assert len(x_forces) == FATABLE_XFA
             assert len(y_forces) == FATABLE_YFA
             assert len(z_forces) == FATABLE_ZFA
 
             self.timestamp = 0
             # those values need to have same name as in SAL/DDS (including
             # capitalization style)
-            self.xForces = x_forces
-            self.yForces = y_forces
-            self.zForces = z_forces
+            self.xForces = x_forces.copy()
+            self.yForces = y_forces.copy()
+            self.zForces = z_forces.copy()
 
             self.fas = fas
 
             self.__calculate_forces_and_moments()
 
+        def clear_quadrants(self, *quadrants: int) -> "ForceCalculator.AppliedForces":
+            """Clear (null) values from given quadrant(s).
+
+            Parameters
+            ----------
+            quadrants : `int...`
```

Comment:
`int...` or `int`?

Reply:
Probably *quadrants : `int` is the best solution.